assets/src/modules/Config.js

```diff
@@ -216,6 +230,42 @@ export class Config {
         return [...this._layersOrder];
     }
 
+    /**
+     * The list of format for file export
+     * @type {string[]}
```

Comment:
```suggestion
     * @returns {string[]}
```
?

Reply:
It is a getter not a method. In this case `@returns` is not valid.

Reply:
Indeed, thanks.
TIL